src/containers/BuildUnsignedSweepCoin/ArbethTokenForm.tsx

```diff
@@ -0,0 +1,163 @@
+    initialValues: {
+      apiKey: '',
+      backupKey: '',
+      backupKeyId: '',
+      gasLimit: 1000000,
+      maxFeePerGas: 20,
+      maxPriorityFeePerGas: 10,
+      recoveryDestination: '',
+      tokenContractAddress: '',
+      userKey: '',
+      userKeyId: '',
+      walletContractAddress: '',
+    },
+    validationSchema,
+  });
+
+  return (
+    <FormikProvider value={formik}>
+      <Form>
+        <h4 className="tw-text-body tw-font-semibold tw-border-b-0.5 tw-border-solid tw-border-gray-700 tw-mb-4">
+          Self-managed cold wallet details
+        </h4>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="Your user public key, as found on your recovery KeyCard."
+            Label="User Public Key"
+            name="userKey"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="Your user Key ID, as found on your KeyCard. Most wallets will not have this and you can leave it blank."
+            Label="User Key ID (optional)"
+            name="userKeyId"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The backup public key for the wallet, as found on your recovery KeyCard."
+            Label="Backup Public Key"
+            name="backupKey"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="Your backup Key ID, as found on your KeyCard. Most wallets will not have this and you can leave it blank."
+            Label="Backup Key ID (optional)"
+            name="backupKeyId"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The ArbethToken address of the wallet contract. This is also the wallet's base address."
+            Label="Wallet Contract Address"
+            name="walletContractAddress"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The address of the smart contract of the token to recover. This is unique to each token, and is NOT your wallet address."
+            Label="Token Contract Address"
+            name="tokenContractAddress"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="The address your recovery transaction will send to."
+            Label="Destination Address"
+            name="recoveryDestination"
+            Width="fill"
+          />
+        </div>
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText="Gas limit for the ArbethToken transaction. The value should be between 400,000 and 20,000,000. The default is 1,000,000 unit of gas."
+            Label="Gas Limit"
```

Comment:
allCoinMetas can be used for minGasLimit=400,000 and defaultGasLimit=1,000,000 value